lard.gemspec

```diff
@@ -1,6 +1,8 @@
+require_relative 'lib/lard'
+
 Gem::Specification.new do |s|
   s.name = 'lard'
-  s.version = '0.0.6'
+  s.version = Lard::VERSION
```

Comment:
I'm a little dubious this will work because of the `require_relative` call above; So I'd like to sacrifice version `0.0.7` to testing this.

Reply:
https://guides.rubygems.org/patterns/#prerelease-gems says I should use pre release gems from now on since I wasted `0.0.7` by forgetting to include `'lib/lard.rb'` into `spec.files` 🙃